templates/unbound.conf.erb

```diff
@@ -147,7 +147,9 @@ server:
 <%= print_config('hide-version', @hide_version) -%>
 <%= print_config('version', @version) -%>
 <%= print_config('hide-trustanchor', @hide_trustanchor, '1.6.2') -%>
-<%= print_config('target-fetch-policy', @target_fetch_policy) -%>
+<%- unless @target_fetch_policy.nil? -%>
```

Comment:
could also keep the type signature the same and check for empty? here, Happy to change i honestly can go either way on theses two styles depending on the day, but i think the current signiture is probably more specific, if a bit verbose.